atomic_reactor/plugins/pre_add_help.py

```diff
@@ -45,13 +48,17 @@ def __init__(self, tasker, workflow, help_file=DEFAULT_HELP_FILENAME):
     def run(self):
         """
         run the plugin
+
+        The plugin returns None if exception occurred,
```

Comment:
What's the benefit of returning these 3 values? Anything inspecting the result of this plugin likely cares whether or not the help file was generated. IMO using `int`s like this will make it harder to debug plugin results.

If we do care about the error cause, I'd much rather this plugin returned something like this:

```python
{
  'help_file': 'my-super-helpful-file',
  'error': None # or ''
}
```

```python
{
  'help_file': None, # or ''
  'error': 'No help file found'
}
```

Having the plugin return the `help_file` that was generated also encapsulates the default name behavior.